### PR TITLE
Fruit branch

### PR DESCRIPTION
**Pull request by Artsapollo**

### CODE REVIEW

File: src/main/java/CollectionsHW/Store.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
+package CollectionsHW;
+
+import java.time.LocalDate;
+import java.util.*;
+
+public class Store {
+    public static List<Fruits> listOfFruits(){
+        List<Fruits> fruitList = new ArrayList<>();
+
```
Comment (Cherkashyn) [inline src/main/java/CollectionsHW/Store.java]: You can use a constructor here, something like: constructor(Fruits ... fruits){ list = Arrays.asList(fruits)}

File: src/main/java/CollectionsHW/Store.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        fruitList.add(dragon);
+        fruitList.add(dragon1);
+        fruitList.add(dragon2);
+
+        return fruitList;
+    }
+
+    public static void freshFruits(List<Fruits> list){
+        for(int i = 0; i < list.size(); i++){
+            if(list.get(i).getDate().isAfter(LocalDate.of(2018, 12, 12))){
```
Comment (Cherkashyn) [inline src/main/java/CollectionsHW/Store.java]: For each looks shorter here

File: src/main/java/CollectionsHW/Store.java [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+                System.out.println("The freshest " + list.get(i));
+            }
+        }
+    }
+
+    public static void fruitsType(List<Fruits> list){
+        for(int i = 0; i < list.size(); i++){
```
Comment (Cherkashyn) [inline src/main/java/CollectionsHW/Store.java]: You should add Type type into method parameters

File: src/main/java/CollectionsHW/Store.java [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+            if(list.get(i).getType().equals(FruitsType.STRAWBERRY) || list.get(i).getType().equals(FruitsType.PINEAPPLE)){
+                System.out.println("Those Dragonfruit still good " + list.get(i));
+            }
+        }
+    }
+
+    public static void expireTime(List<Fruits> list){
```
Comment (Cherkashyn) [inline src/main/java/CollectionsHW/Store.java]: Same. expiration date should be a parameter

File: src/main/java/CollectionsHW/Market.java [hunk 5]
```diff
@@ -0,0 +1,19 @@
+package CollectionsHW;
+
+import java.util.List;
+
+public class Market extends Store{
+    public static void main(String[] args) {
+        List<Fruits> fruttis = listOfFruits();
```
Comment (Cherkashyn) [inline src/main/java/CollectionsHW/Market.java]: How did you assign method to variable List<Fruits> fruttis? Something wrong here.

Comment (Artsapollo) [reply src/main/java/CollectionsHW/Market.java]: listOfFruits() - returns list of fruits which I created in that method from Store.class

File: src/main/java/CollectionsHW/FruitsType.java [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+        return country;
+    }
+}
+
+/*
+растоппить чайную ложку масла
+две столовых ложки сахара
+три ст.ложки сметаны
+две ст.л какао
+без горки всё
```
Comment (SergeyPasko) [inline src/main/java/CollectionsHW/FruitsType.java]: Hm

File: src/main/java/CollectionsHW/Store.java (hunk 1 above)
Comment (SergeyPasko) [reply src/main/java/CollectionsHW/Store.java]: Yes it better way
You don't need always send to methods List<Fruits> as param

File: src/main/java/CollectionsHW/Store.java [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+
+    public static void fruitsType(List<Fruits> list) {
+        for (Fruits find : list) {
+            if (find.getType().equals(FruitsType.STRAWBERRY) || find.getType().equals(FruitsType.PINEAPPLE)) {
+                System.out.println("Those Dragonfruit still good " + find);
+            }
+        }
+    }
+
+    public static void expireTime(List<Fruits> list) {
```
Comment (SergeyPasko) [inline src/main/java/CollectionsHW/Store.java]: Read my home tasks. You must set value as param. Magic number it isn't good. Same to others

File: src/main/java/CollectionsHW/Store.java [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+import java.util.*;
+
+public class Store {
+    private List<Fruits> fruitsList;
+
+    public Store(Fruits... fruits) {
+        fruitsList = Arrays.asList(fruits);
+    }
+
+    public void freshFruits(LocalDate date) {
```
Comment (SergeyPasko) [inline src/main/java/CollectionsHW/Store.java]: Return list as result. This is result you can use in other places. Now it impossible

Comment (SergeyPasko) [reply src/main/java/CollectionsHW/Store.java]: Same to others

File: src/main/java/CollectionsHW/Store.java [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+            if (find.getType().equals(type) && find.getDate().isAfter(date)) {
+                list.add(find);
+            }
+        }
+        System.out.println("-------------------------------------------------------------------------------------------------");
+        System.out.println("Those are still good " + list.toString());
+        return list;
+    }
+
+    public List<Fruits> letsCutThePrice(FruitsType type, Integer expire, Double discount) {
```
Comment (SergeyPasko) [inline src/main/java/CollectionsHW/Store.java]: Read homeTask. Якщо срок реалізації менше певного часу (параметр) то змінити ціну (параметр процент), але це стосується тільки певних видів фруктів. It means FruitType can be many. List\<FruitType\> or FruitType[] or FruitType...

Comment (SergeyPasko) [reply src/main/java/CollectionsHW/Store.java]: One time I set Ok, but in future I will reject

**Pull request merged**